src/command_modules/azure-cli-configure/azure/cli/command_modules/configure/custom.py

```diff
@@ -153,3 +154,72 @@ def _normalize_config_value(value):
     if value:
         value = '' if value in ["''", '""'] else value
     return value
+
+
+def list_cache_contents(cmd):
+    from azure.cli.core.commands.client_factory import get_subscription_id
+    from azure.cli.core._environment import get_config_dir
+    from glob import glob
+    cloud = cmd.cli_ctx.cloud.name
+    subscription = get_subscription_id(cmd.cli_ctx)
+    directory = os.path.join(
+        get_config_dir(),
+        'object_cache',
+        cmd.cli_ctx.cloud.name,
+        get_subscription_id(cmd.cli_ctx))
```

Comment:
https://github.com/Azure/azure-cli/blob/8a83e891fad24f906a8a5fababaa80a5bf6069a1/src/command_modules/azure-cli-configure/azure/cli/command_modules/configure/custom.py#L160-L169

These code are duplicated 3 times in the function. Can we create a _get_deirectory function to do that?

Reply:
Great point! Will do